ome_zarr/bioformats2raw.py

```diff
@@ -0,0 +1,99 @@
+"""
+Spec definitions
+"""
+
+import logging
+import os
+import re
+import tempfile
+from xml.etree import ElementTree as ET
+
+import ome_types
+from ome_zarr_metadata import __version__  # noqa
+
+from ome_zarr.io import ZarrLocation
+from ome_zarr.reader import Node
+from ome_zarr.reader import Spec as Base
+
+__author__ = "Open Microscopy Environment (OME)"
+__copyright__ = "Open Microscopy Environment (OME)"
+__license__ = "BSD-2-Clause"
+
+_logger = logging.getLogger(__name__)
+
+
+class bioformats2raw(Base):
+    @staticmethod
+    def matches(zarr: ZarrLocation) -> bool:
+        layout = zarr.root_attrs.get("bioformats2raw.layout", None)
+        return layout == 3
+
+    def __init__(self, node: Node) -> None:
+        super().__init__(node)
+        try:
+            data = self.handle(node)
+            if data.plates:
+                _logger.info("Plates detected. Skipping implicit loading")
+            else:
+                for idx, image in enumerate(data.images):
+                    series = node.zarr.create(str(idx))
+                    assert series.exists(), f"{series} is missing"
+                    _logger.info("found %s", series)
+                    subnode = node.add(series)
+                    if subnode:
+                        subnode.metadata["ome-xml:index"] = idx
```

Comment:
This is probably the biggest lingering question I have since this basically becomes public API. Thoughts welcome. cc: @will-moore @sbesson _et al._

Reply:
I wonder where this public API is documented?
This new module should be added to https://github.com/ome/ome-zarr-py/blob/master/docs/source/api.rst but I think these changes in parsing also need to be described elsewhere, maybe at https://ome-zarr.readthedocs.io/en/stable/python.html#reading-ome-ngff-images